DQMOffline/LumiZCounting/data/HLTFile_phys14

```diff
@@ -0,0 +1,98 @@
+// SingleMu
```

Comment:
The files in data/ and the data/ subfolder will be removed. Actually the trigger menu is not needed, since we plan to use one same trigger over entire data taking period.